client/components/payment-methods-list/payment-method.tsx

```diff
@@ -319,13 +319,13 @@ const PaymentMethod = ( {
 												'Base transaction fees: %s',
 												'woocommerce-payments'
 											),
-											formatMethodFeesDescription(
+											formatMethodMinimalFee(
```

Comment:
Will break injection of ToS link into a pill for incentives.

Reply:
I encourage you to look into `// Some payment methods doesn't have base percentage rate. In this case, the lowest rate will be shown as a start value` comment, that could point to the bug source.